peer.go

```diff
@@ -1336,17 +1336,19 @@ func messageSummary(msg lnwire.Message) string {
 			msg.Complete)
 
 	case *lnwire.ReplyChannelRange:
-		return fmt.Sprintf("complete=%v, encoding=%v, num_chans=%v",
-			msg.Complete, msg.EncodingType, len(msg.ShortChanIDs))
+		return fmt.Sprintf("start_height=%v, end_height=%v, "+
+			"num_chans=%v, encoding=%v", msg.FirstBlockHeight,
+			msg.LastBlockHeight(), len(msg.ShortChanIDs),
+			msg.EncodingType)
 
 	case *lnwire.QueryShortChanIDs:
 		return fmt.Sprintf("chain_hash=%v, encoding=%v, num_chans=%v",
 			msg.ChainHash, msg.EncodingType, len(msg.ShortChanIDs))
 
 	case *lnwire.QueryChannelRange:
 		return fmt.Sprintf("chain_hash=%v, start_height=%v, "+
-			"num_blocks=%v", msg.ChainHash, msg.FirstBlockHeight,
-			msg.NumBlocks)
+			"end_height=%v", msg.ChainHash, msg.FirstBlockHeight,
```

Comment:
👍

Reply:
Actually, what does this and the above look like for older nodes that send `math.MaxUint32`?

Reply:
It overflows, should be fixed now.